fixturama.gemspec

```
@@ -13,7 +13,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.2"
 
-  gem.add_runtime_dependency "factory_bot", "~> 4.0"
+  gem.add_runtime_dependency "factory_bot", "~> 5.0"
```

I think this would be safer (for those who still uses v4):

```suggestion
  gem.add_runtime_dependency "factory_bot", "> 4.0", "< 6.0"
```